src/dmd/expressionsem.d

```diff
@@ -1705,6 +1705,18 @@ private bool functionParameters(const ref Loc loc, Scope* sc,
             fd.functionSemantic3();
         }
     }
+
+    /* If calling a pragma(inline, true) function,
+     * set flag to later scan for inlines.
+     */
+    if (fd && fd.inlining == PINLINE.always)
+    {
+        if (sc._module)
+            sc._module.hasAlwaysInlines = true;
+        if (sc.func)
+            sc.func.hasAlwaysInlines = true;
+    }
```

Comment:
Why is the check done here?

Reply:
Because `pragma(inline)` may be in the function body and not detected until semantic analysis is done.

Reply:
I was referring to the actual location because `functionParameters` specifically deals with parameters. That doesn't seem to be the right place to deal with inlining.

Reply:
Look up a few lines, where functionSemantic() and functionSemantic3() are done. The pragma can appear in various places, I wanted to pick that up.